R/models.R

```diff
@@ -122,15 +134,26 @@ get_recipie_configurable <- function(train_data,
     }
   }
 
+  rm_lincomb_fn <- function(df) {
+    if (lincomb) {
+      df %>%
+        recipes::step_lincomb(recipes::all_numeric_predictors())
+    } else {
+      df
+    }
+  }
+
   recipes::recipe(Target ~ ., data = train_data %>% dplyr::select(-Combo)) %>%
     mutate_adj_half_fn() %>%
-    step_nz_fn() %>%
```

Comment:
why move the zero variance filter to the end? Wouldn't it benefit to do it first before applying things like PCA?